index.html

```diff
@@ -44,8 +44,181 @@ <h2>LETS CONNECT</h2>
+                <h2>Multi-Post Stories</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/portfolio3.svg" alt="image" />
+                <h2>Tonic</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/portfolio4.svg" alt="image4" />
+                <h2>Multi-Post Stories</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </section>
+        <section class="section3">
+            <div class="about">
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product,
+                    feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+
+                <div class="social">
+                    <p>LET’S CONNECT</p>
```

Comment:
To follow the Figma design, please change your 'LET'S CONNECT' text color to #7f8cff and give the correct spacing.
Figma design:
![image](https://user-images.githubusercontent.com/42799579/193400036-55822498-2c2b-4b6f-a7dc-60a25c61800c.png)

Your design:
![image](https://user-images.githubusercontent.com/42799579/193400066-0d41eaa4-c0e4-4609-8392-fe773706e2e1.png)